### ISSUE TITLE
Cannot match blank indicator

### ISSUE THREAD

**Issue by nichtich** [LibreCat/Catmandu-MARC#107]
How do you select a field with blank indicator?

**Comment by phochste** [comment]
You need to add quotes to map indicators. Catmandu is aggresive trying to interpret strings firsts as JSON paths.

E.g.

```
marc_map("300[ , ]a",test)
retain(test)
```

**Comment by nichtich** [comment]
Nope, this does not work. I've added a test in [branch issue-107](https://github.com/LibreCat/Catmandu-MARC/compare/issue-107). I'd prefer to use `_` and `#` as alias for blank indicator but this likely clashes with #109.

**Comment by hemmefelix** [comment]
For some reason matching on blank indicators now works with Catmandu version 1.2020. Our use case is to only map an ISIL's if a title has not been retracted from a package using this ISIL, which is indicated in MARC field `912 $a` with indicator 1 being present.

```
marc_map('912[ , ]ab','identifier\.packageid.$append')
join_field('identifier\.packageid','|')
replace_all('identifier\.packageid','(ZDB-[\d]{1,4}-[A-Za-z]{1,})([\d]{4})','$1 \($2\)')
```

does extract ZDB-2-SEB (2025) and ZDB-2-SWI (2025), but not ZDB-2-SWI (2026), because the latter is using indicator 1 = 0, see https://unapi.k10plus.de/?id=ebooks:ppn:122409418&format=marcxml for reference
